src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -144,20 +200,32 @@
  (fn [{:keys [db]} [{:keys [message]}]]
    (let [{:keys [offset ownershipStatus collectibles
                  hasMore]} (transforms/json->clj message)
+         ownershipStatus   (update-keys ownershipStatus name)
```

Comment:
It's not a great idea to shadow vars 👀

In general this is an avoided practice in Clojure